src/creativebuilder/domain/services/image_resize_adapter.py

```diff
@@ -0,0 +1,25 @@
+from creativebuilder.constants import CDN_CLOUDFRONT_ROOT, CDN_CLOUDFRONT_IMAGE_RESIZE
+
+from furl import furl
+import urllib
+
+
+class ImageResizeAdapter(object):
```

Comment:
```suggestion
class ImageResizeAdapter:
```
py3에서는 class가 항상 object를 inherit합니다

fyi: https://www.python.org/doc/newstyle/